frontend/src/components/dashboard/aliases/AliasDeletionButton.module.scss

```diff
@@ -0,0 +1,116 @@
+@import "~@mozilla-protocol/core/protocol/css/includes/tokens/dist/index.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "~@mozilla-protocol/core/protocol/css/includes/forms/lib";
+
+.deletionButton {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  background-color: $color-light-gray-30;
+  padding: $spacing-sm $spacing-2xl;
+  border-radius: $border-radius-sm;
+  border: 2px solid transparent;
+  // Keep some space for the focus outline
+  margin: 2px;
```

Comment:
Hmm, so this is the exact width of the outline that wraps this button when it is focused. However, I can't find where that is defined :/ (I think it comes from Protocol. It's visible if I enable the ":focus-visible" pseudo class in my dev tools, but I don't see any relevant rules appearing in the inspector.)

![image](https://user-images.githubusercontent.com/4251/152796232-557248ca-8960-49c2-9c02-b7ee57c623bd.png)

I'd want to make sure I used the same value, to avoid `$spacing-xs` ever getting a different value from whatever the focus outline is, and the layout then breaking.